scripts/autoclose-issues/autoclose.rules.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * @file Auto-close rules for closing won't add icons.
+ */
+
+const autocloseTerm = 'This issue was automatically closed. Please refer to ';
+
+const rules = [
+	{
+		patterns: [/matlab/i],
+		reason: autocloseTerm + '#1233.',
+	},
+	{
+		patterns: [/disney/i],
+		reason: autocloseTerm + '#2309.',
+	},
+	{
+		patterns: [/oracle/i, /java\s/i, /java$/i],
```

Comment:
```suggestion
	{
	        patterns: [/bp/i, /british\s?petroleum/i],
	        reason: autocloseTerm + '#5849.',
	},
	{
	        patterns: [/mattel/i, /barbie/i, /hot\s?wheels/i, /fisher(?:\s|-)price/i],
	        reason: autocloseTerm + '#6656.',
	},
	{
	        patterns: [/microchip/i],
	        reason: autocloseTerm + '#9373.',
	},
	{
		patterns: [/disney/i, /disney\s?(?:plus|+)/i],
		reason: autocloseTerm + '#2309.',
	},
	{
		patterns: [/oracle/i, /java(?:\s|$)/i],
```

Reply:
The `/bp/` pattern may accidentally close other brand requests. We can use `/british\s?petroleum/i` only.